portfolio/src/main/webapp/background.html

```diff
@@ -90,8 +90,8 @@ <h1>Nana's Portfolio</h1>
 
             <br>
 
-            <button onclick="getData()">Click Here For Name</button>
-            <h1 id="data"></h1>
+            <button onclick="addRandomGreeting()">Click Here For Random Quotes</button>
```

Comment:
Eventually consider changing this button label to be something like "addRandomQuote". Refactoring can be painful but it saves you in the future when you want to piece things back together (or use it for future projects!)

Reply:
Thanks! I forgot about that.